public/css/admin-application-review.css

```diff
@@ -127,7 +141,19 @@ input {
 	.application p {
 		font-size: 4vw;
```

Comment:
Same thing about vw font sizes. In this case, it doesn't work because the text becomes uncomfortably large just after the mobile breakpoint. Again, just set a static px height